client/blocks/login/login-form.jsx

```diff
@@ -145,23 +151,25 @@ export class LoginForm extends Component {
 		if ( this.state.isDisabledWhileLoading || this.props.isFormDisabled ) {
 			isDisabled.disabled = true;
 		}
-
+		const errorWithEmail = this.getErrorWithEmailFromCreateErrors();
 		const { requestError } = this.props;
+		const linkingSocialUser = this.isLinkingSocialUser();
+		const usernameOrEmail = this.state.usernameOrEmail || errorWithEmail ? errorWithEmail.error.email : '';
```

Comment:
Same issue as above with `( this.state.usernameOrEmail || errorWithEmail ) ? errorWithEmail.error.email : '';` vs `this.state.usernameOrEmail || ( errorWithEmail ? errorWithEmail.error.email : '' );`. The only difference: This bug interrupts/breaks React's render cycle.